blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
The images are very big (over 7MB in total). Could you compress and replace them? You can find a couple of suggestions for online compressors [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).